src/features/content-explorer/content-explorer/ContentExplorerBreadcrumbs.js

```diff
@@ -28,23 +28,25 @@ const ContentExplorerBreadcrumbs = ({
         >
             <IconChevron direction="left" size="6px" color="#333" />
         </Button>
-        <Breadcrumb label={formatMessage(messages.breadcrumb)}>
+        <Breadcrumb label={formatMessage(messages.breadcrumb)} {...breadcrumbProps}>
             {foldersPath.map((folder, i) => (
-                <PlainButton
+                <div
```

Comment:
Just wanted to confirm...does this affect A11Y? Since it's going from a `button` rendered by `<PlainButton>` to a `div`?

Reply:
I changed it so it doesn't. We still use a Button, but applied styling to a div wrapper